src/app/features/email-template-selection-dialog/email-template-selection-dialog.component.html

```diff
@@ -0,0 +1,50 @@
+<mat-dialog-content>
+  <h2 i18n>Select Email Template</h2>
+  <p>
+    <span i18n>
+      Select the template to use for sending an email to this record. If you
+      want to create a new template, you can do so from -
+    </span>
+    <a
+      [routerLink]="EmailTemplate.route"
+      *appDisabledEntityOperation="{
+        entity: EmailTemplate,
+        operation: 'update',
+      }"
+      matDialogClose
+      i18n
+      >configure available Email templates</a
+    >
+  </p>
+  <p>
+    <span i18n>
+      Note: Nothing will happen unless you have a default email client set up.
+      Please configure one to proceed.
+    </span>
+  </p>
+
+  <app-entity-select
```

Comment:
My password manager tries to fill this (and overlaps our selection dropdown ...).

Maybe we can add `autocomplete="off"` to the input html in the entity-select component in general?
https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/Turning_off_form_autocompletion

Reply:
I think then we should add this into the basicautocomplete component because it's been used in entity-select

Reply:
I tried to add this, but still the password extension is overlapping. I found that Browsers often ignore both autocomplete="off" and autocomplete="false" on input fields, especially if the input name, id, or context resembles an email, password, or credential field. This triggers the browser’s password manager popup regardless of your intentions.